com.invoiceninja.app.yml

```diff
@@ -0,0 +1,35 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/flatpak/flatpak-builder/main/data/flatpak-manifest.schema.json
+
+---
+app-id: com.invoiceninja.app
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+command: invoiceninja
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+  - --filesystem=xdg-pictures
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-download
+modules:
+  - name: Invoice-Ninja
+    buildsystem: simple
+    only-arches:
+      - x86_64
+    build-commands:
+      - "./build-flatpak.sh"
+    sources:
+      - type: file
+        url: https://github.com/invoiceninja/admin-portal/releases/download/v5.0.128/Invoice-Ninja-Linux-Portable.tar.gz
+        sha256: 8275387f9f71a3c81df5f9932796dc23565fd7e4a56ab8705b1e214f86791511
+      - type: git
+        url: https://github.com/invoiceninja/admin-portal.git
+        branch: master
```

Comment:
NEVER a git branch. you need a tag or a commit.

Reply:
Noted: I'll update it

Reply:
This has been corrected